spec.v1.yml

```diff
@@ -189,26 +166,49 @@ paths:
           required: true
           schema:
             type: string
+        - in: path
+          name: bookNumber
+          description: number of the book
+          required: true
+          schema:
+            type: string
         - $ref: "#/components/parameters/limit"
         - $ref: "#/components/parameters/page"
-  "/collections/{collectionName}":
+  "/collections/{collectionName}/hadiths/{hadithNumber}":
     get:
-      summary: Get collection by name
+      summary: Get a hadith of a collection
       description: ""
       responses:
         "200":
-          description: A collection
+          description: Hadith of a book of a collection
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/Collection"
+                $ref: "#/components/schemas/Hadith"
       parameters:
         - in: path
           name: collectionName
           description: name of collection
           required: true
           schema:
             type: string
+        - in: path
+          name: hadithNumber
+          description: number of the hadith
```

Comment:
Please capitalize the first letter in the `summary` and `description` fields (not just here but consistently throughout the file). Also, lowercase Collection and Book when they occur in the middle of a sentence in those fields.

Reply:
Done